polkadot-parachains/res/encointer-kusama.json

```diff
@@ -1,7 +1,9 @@
 {
   "bootNodes": [
     "/ip4/104.248.131.111/tcp/30333/p2p/12D3KooWGa6QRfYmeCWCBHPvJAytaiXZsWzuq1L7zgzph4YHwGjB",
-    "/ip4/104.248.135.198/tcp/30333/p2p/12D3KooWCSzi8TmqvxKmrPdXT5jietrRMCeDVQGZUWQ5JT6MpZ9F"
+    "/ip4/104.248.135.198/tcp/30333/p2p/12D3KooWCSzi8TmqvxKmrPdXT5jietrRMCeDVQGZUWQ5JT6MpZ9F",
+    "/dns/boot.stake.plus/tcp/36333/p2p/12D3KooWNFFdJFV21haDiSdPJ1EnGmv6pa2TgB81Cvu7Y96hjTAu",
```

Comment:
seems to be down?

Reply:
shows 0 peers:
`/dns/boot.stake.plus/tcp/36333/p2p/12D3KooWNFFdJFV21haDiSdPJ1EnGmv6pa2TgB81Cvu7Y96hjTAu`

Reply:
If you test with --reserved-only it will not show peers because you're only allowing the client to connect to reserved nodes and the only reserved node defined is our node. The node is currently online with 19 peers. The best way to test is to remove all the bootnodes from the configured json, compile, and then test using the --bootnode flag. It will only connect to the peer defined and will yield peers without the reserved-only flag. 

May 16 07:45:49 encointerKusama-bootnode encointer-n1[2662240]: 2023-05-16 07:45:49 [Parachain] ðŸ’¤ Idle (19 peers), best: #2919987 (0xd3b5â€¦4b7b), finalized #2919985 (0xb53fâ€¦cfcb), â¬‡ 18.3kiB/s â¬† 4.5kiB/s
May 16 07:45:51 encointerKusama-bootnode encointer-n1[2662240]: 2023-05-16 07:45:51 [Relaychain] ðŸ’¤ Idle (15 peers), best: #17937023 (0x7b08â€¦44a4), finalized #17937020 (0xdd55â€¦a5f4), â¬‡ 180.6kiB/s â¬† 248.7kiB/s

Reply:
the test that you suggest is way too involved for our status quo. If I use the IP bootnode, it works. it syncs and shows me 1 peer. And other, like #166 worked too - also with dns. Can you explain why it doesn't work in your case? what can be reasons?

Reply:
```
2023-05-16 14:10:00 [Relaychain] error=Sender(AskActiveDisputesCanceled) ctx="on FromOrchestra"
2023-05-16 14:10:00 [Parachain] Reserved peer 12D3KooWNFFdJFV21haDiSdPJ1EnGmv6pa2TgB81Cvu7Y96hjTAu disconnected    
2023-05-16 14:10:00 [Parachain] 🔍 Discovered new external address for our node: /ip4/178.197.217.217/tcp/30333/ws/p2p/12D3KooWEk9GBf35PvuSrYJip9RmELDhH7B5o1UwPoaC9G8HDp7f    
2023-05-16 14:10:04 [Relaychain] ⚙️  Syncing, target=#17937266 (24 peers), best: #3776 (0x6b6a…0cb9), finalized #3585 (0x6b4a…355f), ⬇ 321.8kiB/s ⬆ 39.4kiB/s    
2023-05-16 14:10:04 [Parachain] 💤 Idle (0 peers), best: #0 (0x7dd9…5b5b), finalized #0 (0x7dd9…5b5b), ⬇ 0.5kiB/s ⬆ 0.5kiB/s    
2023-05-16 14:10:08 [Parachain] Reserved peer 12D3KooWNFFdJFV21haDiSdPJ1EnGmv6pa2TgB81Cvu7Y96hjTAu disconnected    
2023-05-16 14:10:09 [Relaychain] ⚙️  Syncing 639.7 bps, target=#17937266 (27 peers), best: #6976 (0xe582…3dbf), finalized #6656 (0x4c7e…aed1), ⬇ 335.6kiB/s ⬆ 56.5kiB/s    
2023-05-16 14:10:09 [Parachain] 💤 Idle (0 peers), best: #0 (0x7dd9…5b5b), finalized #0 (0x7dd9…5b5b), ⬇ 0.2kiB/s ⬆ 0.2kiB/s    
2023-05-16 14:10:14 [Relaychain] ⚙️  Syncing 588.5 bps, target=#17937266 (41 peers), best: #9920 (0x710d…ec68), finalized #9728 (0x6862…9aea), ⬇ 331.8kiB/s ⬆ 36.6kiB/s    
2023-05-16 14:10:14 [Parachain] 💤 Idle (0 peers), best: #0 (0x7dd9…5b5b), finalized #0 (0x7dd9…5b5b), ⬇ 68 B/s ⬆ 78 B/s    
2023-05-16 14:10:16 [Parachain] Reserved peer 12D3KooWNFFdJFV21haDiSdPJ1EnGmv6pa2TgB81Cvu7Y96hjTAu disconnected    
2023-05-16 14:10:19 [Relaychain] ⚙️  Syncing 260.6 bps, target=#17937267 (50 peers), best: #11224 (0x5dfe…2cc9), finalized #10752 (0x0877…1abf), ⬇ 348.0kiB/s ⬆ 39.4kiB/s    
2023-05-16 14:10:19 [Parachain] 💤 Idle (0 peers), best: #0 (0x7dd9…5b5b), finalized #0 (0x7dd9…5b5b), ⬇ 0.2kiB/s ⬆ 0.2kiB/s    
2023-05-16 14:10:23 failed to associate send_message response to the sender    
2023-05-16 14:10:23 failed to associate send_message response to the sender    
2023-05-16 14:10:23 [Parachain] Reserved peer 12D3KooWNFFdJFV21haDiSdPJ1EnGmv6pa2TgB81Cvu7Y96hjTAu disconnected    
2023-05-16 14:10:24 [Parachain] 💤 Idle (0 peers), best: #0 (0x7dd9…5b5b), finalized #0 (0x7dd9…5b5b), ⬇ 0.2kiB/s ⬆ 0.2kiB/s    
2023-05-16 14:10:24 [Relaychain] ⚙️  Syncing 442.9 bps, target=#17937267 (50 peers), best: #13440 (0xd735…f5f6), finalized #12800 (0x0c36…a657), ⬇ 367.0kiB/s ⬆ 113.8kiB/s    
2023-05-16 14:10:25 failed to associate send_message response to the sender    
2023-05-16 14:10:29 [Parachain] ⚙️  Syncing  0.0 bps, target=#2920101 (1 peers), best: #0 (0x7dd9…5b5b), finalized #0 (0x7dd9…5b5b), ⬇ 76 B/s ⬆ 0.1kiB/s    
2023-05-16 14:10:29 [Relaychain] ⚙️  Syncing 896.0 bps, target=#17937267 (50 peers), best: #17920 (0x3fed…f541), finalized #17920 (0x3fed…f541), ⬇ 524.3kiB/s ⬆ 157.1kiB/s    
2023-05-16 14:10:29 [Parachain] Reserved peer 12D3KooWNFFdJFV21haDiSdPJ1EnGmv6pa2TgB81Cvu7Y96hjTAu disconnected    
2023-05-16 14:10:33 failed to associate send_message response to the sender    
2023-05-16 14:10:33 failed to associate send_message response to the sender    
2023-05-16 14:10:33 failed to associate send_message response to the sender    
```
the same happens if I try a wrong node identity, so, are you sure the key of your bootnode is still correct?